### PR TITLE
Optimize by run single list --format=xml

### PR DESCRIPTION
**Pull request by andytson-inviqa**
The existing code is very slow with more than a couple of commands (especially so on a Docker for Mac osxfs host volume mount).

The symfony/console library supports fetching a single combined xml of all commands, so using this reduces the number of console command runs to one, massively speeding it up.

### CODE REVIEW

File: src/DumpCommand.php [hunk 1]
```diff
@@ -73,42 +73,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+        $process = new Process($script . ' ' . $scriptOptions . ' list --format=xml');
         $process->run();
         if (!$process->isSuccessful()) {
             throw new \RuntimeException($process->getErrorOutput());
         }
 
-        $rawCommands = explode("\n", $process->getOutput());
-        array_pop($rawCommands);
+        $xmlCommands = $process->getOutput();
+        $xml = simplexml_load_string($xmlCommands);
```
Comment (rquadling) [inline src/DumpCommand.php]: simplexml MAY be disabled (http://www.php.net/manual/en/simplexml.installation.php using the `--disable-simplexml` option). Adding this extension as a requirement in `composer.json` would be the easiest way to deal with that. Considering this extension is enabled by default, I'd say that this is a safe solution. (http://www.php.net/manual/en/simplexml.installation.php#121170)

Comment (andytson-inviqa) [reply src/DumpCommand.php]: sure, would be good to add that to the composer.json. Just a note though that this extension was already being used, but I'm happy to add it anyway

Comment (rquadling) [reply src/DumpCommand.php]: Ha! So it was already missing. Hopefully this will be accepted and merged soon.

**Comment by bamarni** [comment]
Thanks, really nice one

**Comment by rquadling** [comment]
Any chance of a release please?

**Comment by rquadling** [comment]
@bamarni Please can you make a release?

**Comment by rquadling** [comment]
@bamarni Does anyone else have release capabilities? It would be really nice for this to be released.

**Pull request merged**